cli/src/cli_output.rs

```diff
@@ -516,19 +516,21 @@ impl fmt::Display for CliStakeState {
             writeln!(f, "Withdraw Authority: {}", authorized.withdrawer)?;
             Ok(())
         }
-        fn show_lockup(f: &mut fmt::Formatter, lockup: &CliLockup) -> fmt::Result {
-            writeln!(
-                f,
-                "Lockup Timestamp: {} (UnixTimestamp: {})",
-                DateTime::<Utc>::from_utc(
-                    NaiveDateTime::from_timestamp(lockup.unix_timestamp, 0),
-                    Utc
-                )
-                .to_rfc3339_opts(SecondsFormat::Secs, true),
-                lockup.unix_timestamp
-            )?;
-            writeln!(f, "Lockup Epoch: {}", lockup.epoch)?;
-            writeln!(f, "Lockup Custodian: {}", lockup.custodian)?;
+        fn show_lockup(f: &mut fmt::Formatter, lockup: Option<&CliLockup>) -> fmt::Result {
+            if let Some(lockup) = lockup {
+                writeln!(
+                    f,
+                    "Lockup Timestamp: {} (UnixTimestamp: {})",
+                    DateTime::<Utc>::from_utc(
+                        NaiveDateTime::from_timestamp(lockup.unix_timestamp, 0),
+                        Utc
+                    )
+                    .to_rfc3339_opts(SecondsFormat::Secs, true),
+                    lockup.unix_timestamp
+                )?;
+                writeln!(f, "Lockup Epoch: {}", lockup.epoch)?;
```

Comment:
We could be a little more-smarter here and only display the timestamp or epoch field if it's not default, which might be less confusing to users when only one is set?

Reply:
I can't argue with more-smarter! 😉

Reply:
📈 🧠 https://github.com/solana-labs/solana/pull/10278/commits/4e03938a4401e3002907cdc21efd9c4d3224071b?w=1#diff-506d0550bb762fa51761fd1784b4c53fR521-R535